Framework/Crystal/src/CentroidPeaks.cpp

```diff
@@ -355,5 +360,48 @@ void CentroidPeaks::removeEdgePeaks(
   peakWS.removePeaks(std::move(badPeaks));
 }
 
+void CentroidPeaks::sizeBanks(const std::string &bankName, int &nCols,
+                              int &nRows) {
+  if (bankName == "None")
```

Comment:
Wouldn't it be easier to tell the nRows by running `compInfo->componentsInSubtree(bankIndex) - compInfo->detectorsInSubtree(bankIndex)`
At least for `RectangularDetectors` that is. For `RectuangularDetectors` I believe that the only non-detectors in the subtree are the `CompAssemblies` forming the columns.

Reply:
Done